chicken-sloths/bangazon-cli-sprint2
# Completes basic payment type prompt

# Description
Basic functionality for closing a customer's ORDER

## Related Ticket(s)
#6 

## Problem to Solve
This solves the problem of not having a basic foundation to work off of for the ability to close a customer's order.

## Proposed Changes
Provides very basic outline of how this prompt should work. `addCustPaymentOptV` defines what the prompt should look like with some basic functionality. Declares the appropriate `OrdersM` & `PaymentOptionsM` methods to be defined later. Outlines how `addCustPaymentOptC` should call its models and `addCustPaymentOptV`.

## Expected Behavior
User should be able to select `5. Complete an order` and be given a prompt. That prompt should send them back to the main menu if they enter `N` for the first prompt. If the user enters `Y`, then they enter in another response in the next prompt before the program closes down.

## Steps to Test Solution
1. `npm start`
1. Select `6`
1. Select `Y`, then type in something for the next prompt, & you should be taken back to the main menu
1. Select `6`, again
1. Select `N`, you should be taken back to the main menu!
